decidim-core/app/helpers/decidim/feature_reference_helper.rb

```diff
@@ -13,7 +13,7 @@ module FeatureReferenceHelper
     def feature_reference(feature, options = {})
       return unless feature.reference
       @reference = feature.reference
-      "<div class='reference #{options[:class]}'>#{localized_reference}</div>".html_safe
+      "<div class='tech-info #{options[:class]}'>#{localized_reference}</div>".html_safe
```

Comment:
Why this change?

Reply:
The class was previously used only to format the proposal reference, now it's used to format the creation date as well. I renamed it that catch up with that change.

Reply:
👍